src/modules/fs.cpp

```diff
@@ -27,7 +27,11 @@ namespace modules {
    * setting up required components
    */
   fs_module::fs_module(const bar_settings& bar, string name_) : timer_module<fs_module>(bar, move(name_)) {
-    m_mountpoints = m_conf.get_list(name(), "mount");
+    m_mountpoints = m_conf.get_list(name(), "mount", {});
+    if (m_mountpoints.empty()) {
+      m_log.warn("No mountpoints specified, using fallback \"/\"");
```

Comment:
A warning seems to excessive to me here. The info level would be more appropriate.

Also, log messages in modules should be prefixed with the module name (you can get it by calling `name()`), see the other log messages in the module.